examples/map/main.go

```diff
@@ -24,7 +25,7 @@ func main() {
 	c := canvas.New(100, 100)
 	ctx := canvas.NewContext(c)
 	draw(ctx)
-	c.SavePNG("out.png", 8.0)
+	c.WriteFile("out.png", rasterizer.PNGWriter(8.0))
```

Comment:
Same

Reply:
With the sample code I posted we could write `rasterizer.PNGWriter(8.0 * canvas.DPM)`